src/uosc/main.lua

```diff
@@ -130,6 +130,9 @@ if options.autoload then mp.commandv('set', 'keep-open-pause', 'no') end
 local intl = require('lib/intl')
 t = intl.t
 
+--[[ CHARACTER CONVERSION ]]
+local char_conv = require('lib/char_conv')
```

Comment:
This shouldn't be `local`. It's accessible in `Menu.lua` because you're declaring `char_conv()` globally inside `char_conv.lua`, so there's no point for the assignment here.

Just do this:

```lua
--[[ INTERNATIONALIZATION ]]
local intl = require('lib/intl')
t = intl.t
require('lib/char_conv')
```

And inside `char_conv.lua`, add `local` in front of every variable and function that doesn't need to be global.